### ISSUE TITLE
Bluetooth connects, then immediately drops...

### ISSUE THREAD

**Issue by MichaelNCoakley** [dokterdok/Continuity-Activation-Tool#149]
Hi All,

Thanks for this amazing project.  The work is astounding!

I'm running a Late 2008 15" MBP, 10.10.2  with the appropriate Broadcom dongle attached.  Compatibility check seems to think all is well, but when I try to pair my iPhone 6+ with the MBP, it pairs, very briefly, then says, Not connected.  Any ideas?  Pics below...

https://lh3.googleusercontent.com/-yNp_CWOf2yE/VOUW1Jf3YNI/AAAAAAAAFTI/27s2kSs1C2A/w1109-h420-no/Screen%2BShot%2B2015-02-18%2Bat%2B4.41.42%2BPM.png

https://lh5.googleusercontent.com/-MmvcuaqHFqE/VOUW1AaJcpI/AAAAAAAAFTE/bNekmlBneSg/w756-h524-no/Screen%2BShot%2B2015-02-18%2Bat%2B4.41.19%2BPM.png

Any help is appreciated.

**Comment by dokterdok** [comment]
Hi, iPhone & Mac pairing issues seem to happen with regular Macs as well (see [here](https://discussions.apple.com/thread/6602307) for example). 
That said, your dongle might benefit from a firmware upgrade that solves the issue, as described in [#122](https://github.com/dokterdok/Continuity-Activation-Tool/issues/122#issuecomment-69090833).

**Comment by MichaelNCoakley** [comment]
Thanks for the quick reply, Dokterdok.

In short, things seem to be working now, but I'm not sure *exactly* why.  I successfully completed the firmware upgrade as prescribed in #122, and was still having the issue of pairing breaking only seconds after the pair.  In reading the majority of post on the other link you suggested, I learned that you do not need to BT link the phone and mac for continuity/handoff to work.  If fact, you shouldn't.  With that knowledge, I "forget" the pair on both devices and things seem to work.  I hope this might help others.  The only thing I seemingly can't get to work is audio, either from the caller or myself, to work when answering the phone via the laptop, but that's another issue...

Thank you so much for your help,

Mike
